tests/dbus/test_dbus.py

```diff
@@ -424,6 +425,10 @@ def test_netplan_dbus_config_apply(self):
         self.assertFalse(os.path.isdir(tmpdir))
 
     def test_netplan_dbus_config_try_cancel(self):
+        # touch self._netplan_try_stamp to signal that 'netplan try' is ready
+        # to take (Accept/Reject) input signals, before the timeout
+        self.addCleanup(os.remove, self._netplan_try_stamp)
```

Comment:
(nitpick) I see this cleanup in a bunch of places and wonder if it would make sense to move it into `setUpTest` (or if half the tests need it and half don't maybe split TestNetplanDBus into two where one does the cleanup in central place and one does not).

Reply:
Moving everything into `<TEST-TMP>/run/netplan/*` which is cleaned up after every test-case, this is not needed anymore.